atlasdb-perf/src/main/java/com/palantir/atlasdb/performance/benchmarks/table/StreamingTable.java

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright 2016 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.performance.benchmarks.table;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.util.Random;
+
+import org.openjdk.jmh.annotations.Level;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.TearDown;
+
+import com.google.common.collect.ImmutableMultimap;
+import com.google.common.collect.Multimap;
+import com.palantir.atlasdb.keyvalue.api.KeyValueService;
+import com.palantir.atlasdb.keyvalue.api.Namespace;
+import com.palantir.atlasdb.keyvalue.api.TableReference;
+import com.palantir.atlasdb.performance.backend.AtlasDbServicesConnector;
+import com.palantir.atlasdb.performance.schema.StreamTestSchema;
+import com.palantir.atlasdb.performance.schema.generated.KeyValueTable;
+import com.palantir.atlasdb.performance.schema.generated.StreamTestTableFactory;
+import com.palantir.atlasdb.performance.schema.generated.ValueStreamStore;
+import com.palantir.atlasdb.services.AtlasDbServices;
+import com.palantir.atlasdb.table.api.ColumnValue;
+import com.palantir.atlasdb.table.description.Schemas;
+import com.palantir.atlasdb.table.generation.ColumnValues;
+import com.palantir.atlasdb.transaction.api.TransactionManager;
+import com.palantir.common.persist.Persistable;
+
+@State(Scope.Benchmark)
+public class StreamingTable {
```

Comment:
+1 for the randomness - "bytes" is only there now so I can check that the code I've written is actually sane.